packages/dashboard-message-bus-client/lib/client.ts

```diff
@@ -0,0 +1,223 @@
+import { DashboardMessageBusClientOptions } from "./types";
```

Comment:
This should probably just be defined in this file, since it's only used in this file and since it's going to be the first thing one looks for when reading this file